core/src/main/java/org/apache/calcite/util/BuiltInMethod.java

```diff
@@ -440,6 +440,8 @@ public enum BuiltInMethod {
   LOG(SqlFunctions.class, "log", long.class, long.class),
   TANH(SqlFunctions.class, "tanh", long.class),
   SINH(SqlFunctions.class, "sinh", long.class),
+  CSC(SqlFunctions.class, "csc", long.class),
```

Comment:
@zoudan if I am not mistaken, this change in BuiltInMethod is not required: these `CSC` and `SEC` are not used anywhere, so they should be removed to avoid confusion.
If we check for example a similar PR: https://github.com/apache/calcite/pull/3130 ; in that case BuiltInMethod .java was not impacted, so I think it should be the same in this case.

Reply:
Yes, you are right, I add these by mistake. I will remove them.

Reply:
updated